rust/main/agents/relayer/src/msg/pending_message.rs

```diff
@@ -65,6 +66,8 @@ pub struct PendingMessage {
     ctx: Arc<MessageContext>,
     status: PendingOperationStatus,
     app_context: Option<String>,
+    #[serde(skip_serializing)]
+    max_message_retries: u32,
```

Comment:
nit: def don't feel strongly but could maybe do `max_retries` as the message is redundant (& to more closely relate this to the `num_retries`

Reply:
updated